src/pages/Checkout/CheckoutPage.vue

```diff
@@ -364,6 +364,14 @@ export default {
 			this.activeLoginDuration = parseInt(_get(data, 'general.activeLoginDuration.value'), 10) || 3600;
 		}
 	},
+	beforeRouteEnter(to, from, next) {
```

Comment:
This navigation guard is called before the router.onReady callback in client-entry.js, so this sync will happen before the vue app is mounted on the client after a server render.